modules/administration/pages/tshoot-packages.adoc

```diff
@@ -19,22 +19,13 @@ If more detailed instructions are required, put them in a "Resolving" procedure:
 . Last step
 ////
 
-Packages can sometimes be locked or taskomatic can experience problems, which creates problems with metadata regeneration.
+Packages on a client can sometimes be locked, which creates problems with metadata regeneration.
```

Comment:
```suggestion
When packages on a client are locked, {productname} Server may not be able to correctly determine the set of applicable patches.
```

Rationale: packages can be locked by users, they do not get locked because some process did. It is a known product limitation that the lock information is not considered by the Server when it determines the set of applicable patches.